graphql/util.go

```diff
@@ -58,40 +58,77 @@ func getGroupedFiles(ctx context.Context, name string, taskID string, execution
 	return &GroupedFiles{TaskName: &name, Files: apiFileList}, nil
 }
 
-func setScheduled(ctx context.Context, url, taskID string, isActive bool) (*restModel.APITask, error) {
+func setManyTasksScheduled(ctx context.Context, url string, isActive bool, taskIDs ...string) ([]*restModel.APITask, error) {
 	usr := mustHaveUser(ctx)
-	t, err := task.FindOneId(taskID)
+	tasks, err := task.FindAll(db.Query(task.ByIds(taskIDs)))
 	if err != nil {
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
-	if t == nil {
-		return nil, ResourceNotFound.Send(ctx, errors.Errorf("task %s not found", taskID).Error())
+	if len(tasks) != len(taskIDs) {
```

Comment:
Should we make a FindAllByIds function that does this logging within it rather than duplicating the logic?

Reply:
Sure